src/todo.ts

```diff
@@ -0,0 +1,99 @@
+import { Item } from './item';
+class TodoApp {
+    /**
+     * Allows the creation and maintenance of a simple todo list
+     */
+
+    public toDoList: Map<number, Item>;
```

Comment:
So this is `public`.  Is this what you really want?  Do you want someone outside of this class to be able to set it to something else?  I don't think you do.